shared/templates/platform_os_linux/ansible.template

```diff
@@ -0,0 +1,12 @@
+{{%- set ansible_os_release_name_cond = 'ansible_distribution == "' + OS_ID_ANSIBLE + '"' -%}}
+{{%- macro ansible_os_release_conditional(specs) -%}}
+  {{%- for spec in specs -%}}
+    ansible_distribution_version is version("{{{ spec.ver }}}", "{{{ spec.op }}}")
+  {{% endfor %}}
+{{%- endmacro -%}}
+{{%- if SPECS|length() >= 1 -%}}
```

Comment:
What is `SPECS` and where is it defined?

Reply:
I am sorry for confusion, I removed this part of the code. It will be brought in later when the support for versions is implemented.